data-raw/example_plots.R

```diff
@@ -0,0 +1,155 @@
+## PROJECT: glitr
+## AUTHOR:  AChafetz | USAID
+## PURPOSE: example plots
+## DATE:    2020-06-01
+
+
+# DEPENDENCIES ------------------------------------------------------------
+
+library(tidyverse)
+library(scales)
+library(glitr)
+library(extrafont)
+
+
+
+# SCATTER PLOT ------------------------------------------------------------
+
+
+  set.seed(81)
+  df_scatter <-
+  tibble(x = sample(150:200, 20, replace = TRUE),
+         y = sample(75:125, 20, replace = TRUE),
+         z = sample(1:10, 20, replace = TRUE))
+
+
+  df_scatter %>%
+    ggplot(aes(x, y, size = z, color = y > 100)) +
+    geom_point() +
+    labs(x = NULL, y = NULL,
+         title = "TITLE",
+         subtitle = "caption/description",
+         caption = "data source") +
+    si_style() +
+    theme(legend.position = "none")
+
+
+
+# LINE CHART --------------------------------------------------------------
+
+
+  df_line <- cascade %>%
+    filter(indicator == "TX_NEW",
+           period_type == "results") %>%
+    mutate(pt_start = case_when(period == min(period) ~ value),
+           pt_end = case_when(period == max(period) ~ value),
+           partner_focal = primepartner == "Sagittarius") %>%
+    arrange(primepartner, period)
+
+  largest_partners <- df_line %>%
+    top_n(5, wt = pt_end) %>%
+    pull(primepartner)
+
+  df_line <- filter(df_line, primepartner %in% largest_partners)
+
+
+
+  df_line %>%
+    ggplot(aes(period, value, group = primepartner, color = partner_focal)) +
+    geom_path(size = 1) +
+    geom_point(aes(y = pt_end), shape = 21, size = 4, stroke = 1.5, fill = "white", na.rm = TRUE) +
+    geom_point(aes(y = pt_start, fill = partner_focal), shape = 21, size = 4, na.rm = TRUE) +
+    geom_text(aes(y = pt_end, label = primepartner), na.rm = TRUE,
+              family = "Source Sans Pro", nudge_x = .4) +
+    labs(x = NULL, y = NULL,
+         title = "TITLE",
+         subtitle = "caption/description",
+         caption = "data source") +
+    scale_x_discrete(expand = expansion(add = 1)) +
+    si_style() +
+    theme(legend.position = "none")
+
+
+# BAR CHART ---------------------------------------------------------------
+
+
+  df_bar <- cascade %>%
+    filter(operatingunit == "Saturn",
+           indicator == "TX_NEW",
+           period == "FY50") %>%
+    group_by(primepartner, period, period_type) %>%
+    summarise(value = sum(value, na.rm = TRUE)) %>%
+    ungroup() %>%
+    spread(period_type, value) %>%
+    mutate(targets_q1 = targets * .25) %>%
+    filter(!is.na(targets_q1))
+
+  set.seed(23)
+
+  agency <-
+    tibble(fundingagency = sample(c("USAID", "CDC"), nrow(df_bar), replace = TRUE))
+
+  df_bar <- df_bar %>%
+    bind_cols(agency, .)
+
+  df_bar %>%
+    ggplot(aes(fct_reorder(primepartner, targets_q1, sum), cumulative)) +
+    geom_col(aes(y = targets_q1), fill = "gray90") +
+    geom_col() +
+    geom_errorbar(aes(ymin = targets_q1, ymax = targets_q1), color = "gray90", size = .9) +
```

Comment:
Wasn't sure what the purpose of this line was. Here is the plot result I'm getting w/ it.
[Plot result](https://github.com/USAID-OHA-SI/glitr/issues/14#issue-633632977)

Reply:
result bars plotted against target error bars 👍